launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -88,12 +88,12 @@ class ForkScaffolder(private val launcher: Launcher) {
                         }
                         define("localInstanceSpUri") {
                             label = "Service Pack URI"
-                            description = "Typically file named 'aem-service-pkg-*.zip'"
+                            description = "[AEM on-prem only!] Typically file named 'aem-service-pkg-*.zip'"
```

Comment:
This is not human-friendly description , I mean using `[` and `]` does not look naturally on dialog